server/subject_transform_test.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2018-2023 The NATS Authors
```

Comment:
If new just 2023.

Reply:
Mostly new, but there are still be a few lines that are from before I started modifying subject transforms (according to git blame on accounts_test.go which is where those came from it looks like they are a few lines that are about 2 years old) want me to change to 2020-2023 instead?

Reply:
Just do 2023.. Thx

Reply:
Done